docs/Data Lake.md

```diff
@@ -128,24 +128,44 @@ After clicking the *Analyze Data* button, the app will read the selected dataset
 
 *   Principal Component Analysis (PCA)
```

Comment:
Bold these and change image260(Remove TRV Data lake).

Reply:
made the changes.